pkg/test/echo/proto/echo.proto

```diff
@@ -90,6 +90,11 @@ message ForwardEchoRequest {
   // HBONE communication settings. If provided, requests will be tunnelled.
   HBONE hbone = 24;
 
+  // S
+  repeated HBONE double_hbone = 27;
```

Comment:
@howardjohn I didn't see any other way to specify different settings for the inner hbone tunnel, but I still don't love it. Any ideas?

Reply:
Does it need to be repeated? Since we are only going to have 2 layers you could just make this represent the outer HBONE and the existing field represent the inner?